routes/student.js

```diff
@@ -0,0 +1,27 @@
+import express from 'express';
+import { 
+  createStudentController, 
+  getAllStudentsController, 
+  getStudentByIdController, 
+  updateStudentController, 
+  deleteStudentController
+} from '../controllers/student.js';
+
+const router = express.Router();
+
+// Create a new student
+router.post('/', createStudentController);
+
+// Get all students
+router.get('/', getAllStudentsController);
+
+// Get student by ID
+router.get('/:id', getStudentByIdController);
+
+// Update student by ID
+router.put('/:id', updateStudentController);
+
+// Delete student by ID
+router.delete('/:id', deleteStudentController);
```

Comment:
```suggestion
router.post('/', createStudentController);
router.get('/', getAllStudentsController);
router.get('/:id', getStudentByIdController);
router.put('/:id', updateStudentController);
router.delete('/:id', deleteStudentController);
```

If you have any specific router-path like `/students/:id/results` or `/students/:id/results/:year` then you have to drop the comment like 
- `/students/:id/results` // fetch all results for a student
- `/students/:id/results/:year` // fetch results for a specific year for a student